### ISSUE TITLE
Firestore User Authentication

### ISSUE THREAD

**Issue by servicesoftuk** [transistorsoft/background-geolocation-firebase#2]
I'm still at the R&D stage and have a prototype app which uses the background-geolocation plugin with the additional background-geolocation-firebase plugin.  GeoLocations are feeding into my Firestore database as expected.

The default Firestore security rules allow read/write to any user, which I will need to resolve before I can complete the R&D project.  If I change the security rules to restrict access to authenticated users, the Firestore plugin can no longer write to the database.

I cannot see any provision for user authentication in the plugin documentation.  Have I missed it or do you have any plans to implement in the future?

**Comment by christocracy** [comment]
No, the plugin is not involved with authentication.  Is there something I need to add?  I thought this could be handled from the server end.
